src/docs/Dockerfile

```diff
@@ -0,0 +1,47 @@
+## Base ########################################################################
+# Use a larger node image to do the build for native deps (e.g., gcc, python)
+FROM node:lts as base
+
+# Reduce npm log spam and colour during install within Docker
+ENV NPM_CONFIG_LOGLEVEL=warn
+ENV NPM_CONFIG_COLOR=false
+
+# We'll run the app as the `node` user, so put it in their home directory
+WORKDIR /home/node/app
+# Install pnpm
+RUN npm install -g pnpm
+# Copy the source code over
+COPY --chown=node:node . /home/node/app/
+
+## Development #################################################################
+# Define a development target that installs devDeps and runs in dev mode
+FROM base as development
+WORKDIR /home/node/app
+# Install (not ci) with dependencies, and for Linux vs. Linux Musl (which we use for -alpine)
+RUN pnpm install
+# Switch to the node user vs. root
+USER node
```

Comment:
You might want to move this up before your `RUN` command so that the files are installed as `node`

Reply:
When I moved `USER node` above `RUN pnpm install`, I get this error:

```
 => [development 1/2] WORKDIR /home/node/app                                                          0.1s 
 => ERROR [development 2/2] RUN pnpm install                                                          1.0s
------
 > [development 2/2] RUN pnpm install:
#10 0.937  EACCES  EACCES: permission denied, open '/home/node/app/_tmp_6_4876fde50687bbcdaee462064b1bd2a0'#10 0.937
#10 0.937
#10 0.937
------
executor failed running [/bin/sh -c pnpm install]: exit code: 1
```

I didn't get an error before the change but what you said 
> You might want to move this up before your RUN command so that the files are installed as node

made sense. I'm not sure why it doesn't work on my machine